### PR TITLE
Cleanup README

### PR DESCRIPTION
**Pull request by toddfast**
Edited for style and consistency. Improve language in favor of clear value proposition and less subjectivity.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,24 +12,23 @@
 
 ## Project
 
-Wasabi A/B Testing Service is a real-time, enterprise-grade, 100% API driven project. Users are empowered to own their own data, and run experiments across web, mobile, and desktop. It’s fast, easy to use, it’s chock full of features, and instrumentation is minimal.
-
-Learn more about how Wasabi can empower your team to move from hunches to actionable, data-driven user insights with our simple, flexible, and scalable experimentation platform.
+Wasabi is a mature, feature-rich, open-source A/B testing platform, complete with an experiment management UI. Wasabi powers all of Intuit's in-product experimentation and has been running in production at scale for several years.
```
Comment (iizrailevsky) [inline README.md]: @toddfast Evaluators might be confused that this platform is Intuit-specific when they see this in the 1st paragraph: "Wasabi powers all of Intuit's in-product experimentation and has been running in production at scale for several years" . Would it make sense to move it to the 2nd paragraph as a proof of scale? [cc: @calvinwma @asidhwani1]

File: README.md [hunk 2]
```diff
@@ -12,24 +12,23 @@
...
 
+Wasabi differs from other A/B testing services and frameworks in two ways. First, Wasabi runs on your servers, giving you full control over your data. Second, as something we call _A/B-Testing-as-a-Service_, Wasabi focuses on API-level integration with your apps and products. This means that Wasabi can be used to run experiments across Web, mobile, and desktop user experiences, as well as experiements on back-end, non-user-facing components, like business process workflows, search and routing algorithms, and predictive models.
```
Comment (iizrailevsky) [inline README.md]: +1 - good differentiation points!

File: README.md [hunk 3]
```diff
@@ -12,24 +12,23 @@
...
 
-* **Own your own data** - Don’t ship your data to 3rd party services. Manage your own data and protect your users to maintain your competitive advantage.
-* **Enterprise Grade** - Wasabi has been battle tested in production in fin-tech for 2 years in TurboTax, QuickBooks, Mint.com, etc., powering over 1,000 experiments across Intuit.
-* **High Performance** - Consistent performance response times within 30ms server side.
-* **100% API Driven** - Compatible with any language and environment.
-* **Platform Agnostic** - A uniform, consistent experience across web, mobile, desktop, and also front-end, back-end.
-* **Get up and running in minutes** - Spin up your Wasabi docker in 5 minutes and be in production with the platform, instrumentation, and experiments within a day.
-* **It’s Free** - Don’t pay an arm and a leg for an enterprise-grade platform – the platform is free! 
-* **Real-time assignments** - Assign users into experiments in real-time, to preserve traffic for other parallel A/B tests.
+* **Own your own data** - Wasabi runs on your servers, in the cloud or on-premise, where you have complete control over your data.
```
Comment (iizrailevsky) [inline README.md]: @calvinwma @asidhwani1 As you were the original authors, any thoughts? Most of the update look good to me

File: README.md [hunk 4]
```diff
@@ -12,24 +12,23 @@
...
+* **Platform Agnostic** - Uniform, consistent testing across Web, mobile, desktop, and also front-end, back-end.
+* **Real-time user assignments** - Assign users into experiments in real time, to preserve traffic for other parallel A/B tests.
 * **Cloud and on-premise** - Designed to live in the cloud or in your own data center.
-* **Analytics** - Core analytics functionality and metrics visualization out of the box. 
-* **Administrative UI** - Manage and setup your experiments using our user friendly interface.
+* **Analytics** - Core experiment analytics and metrics visualization out of the box, as well as the ability to send data to your existing analytics infrastructure.
+* **Pluggable** - Well-defined interfaces for plugging in your own access control, sending data to data pipelines, and providing fully custom bucket allocations.
+* **Experiment Management UI** - Setup and manage experiments via a modern Web interface. Management via REST API is also possible.
+* **Dockerized** - Spin up a Wasabi Docker instance in 5 minutes and be in production with the platform, instrumentation, and experiments within a day.
```
Comment (iizrailevsky) [inline README.md]: @toddfast It's more like 15 min with VM docker now. But it's a lot fast (more like 5 min) with native docker on Mac (work in progress) :)

File: README.md [hunk 5]
```diff
@@ -44,8 +43,7 @@ Learn more about how Wasabi can empower your team to move from hunches to action
 
 ## Get Started
 
-Installing Wasabi is straightforward. The following steps will install the needed tools, build and run a
-complete stack noting that at this time only OSX is supported.
+The following steps will help you install the needed tools, then build and run a complete Wasabi stack. Note, at this time, only Mac OS X is supported.
```
Comment (iizrailevsky) [inline README.md]: 👍 on the rest of the updates!

**Comment by iizrailevsky** [comment]
@toddfast Thanks for reviewing the README and putting the effort in updates! Good with most of them, with the exception of a few (see the comments).
@asidhwani1 @calvinwma Please review - especially the initial features section

File: README.md (hunk 1 above)
Comment (calvinwma) [inline README.md]: I too am worried about users drawing conclusions about Wasabi being Intuit centric.  Perhaps we remove that line and combine with the following paragraph and let it be highlighted in the features section?

File: README.md (hunk 3 above)
Comment (calvinwma) [inline README.md]: I like highlighting Wasabi being free but it is implied as it is open source and am ok with removing it.

File: README.md (hunk 4 above)
Comment (calvinwma) [inline README.md]: I'd think that we should leave the UI feature there as it is something that provides value to the customer

Comment (jwtodd) [inline README.md]: more like 20s once warmed up ... see #8

File: README.md [hunk 6]
```diff
@@ -296,17 +289,15 @@ development: {
 % ./bin/wasabi.sh stop
 ```
 
-Now while that was fun, in all likelihood you will be using an IDE to develop Wasabi features. In doing so you need only
-add the aforementioned configuration information to your Wasabi JVM runtime prior to startup:
+Now while that was fun, in all likelihood you will be using an IDE to work on Wasabi. In doing so, you need only
+add the configuration information above to the JVM commandline prior to startup:
```
Comment (jwtodd) [inline README.md]: ?above?

**Pull request closed without merging**